File: JavaProject/I_Manager.java
import java.util.Date;
public interface I_Manager {
	public void View_reports();
	public void Print_and_archive_report(Date date);
	public void Add_daily_report();
	public void PayOut_salary(String employee_ID);
	public void Give_bonus(String employee_id, double bonus);
	public void Calculate_salary(String employee_id);
	public void Fire_employee(String employee_id);
	public void Check_hotel_statistics();
	public void Call_employee(String employee_id);
	public void Warn_employee(String employee_id);
}

File: CSharpProject/Hotel/Model/IManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hotel.Model
{
    interface IManager
    {
        void ViewReports();
        void PrintAndArchiveReport(DateTime date);
        void AddDailyReport();
        void PayOutSalary(String employeeID);
        void GiveBonus(String employeeID, double bonus);
        void CalculateSalary(String employeeID);
        void FireEmployee(String employeeID);
        void CallEmployee(String employeeID);
        void WarnEmployee(String employeeID);
    }
}
